ivy/functional/frontends/tensorflow/tensor.py

```diff
@@ -53,6 +53,9 @@ def shape(self):
     def get_shape(self):
         return tf_frontend.raw_ops.Shape(input=self)
 
+    def ref(self):
+        return self._ivy_array.ref()
+
     def set_shape(self, shape):
         if shape is None:
             return
```

Comment:
def ref(self, shape):
      if shape is None:
           return self._ivy_array.ref()

Reply:
Hey @Kamalesh3112 
Thanks for reviewing! :)

But, according to the TensorFlow API docs, `ref` method doesn't have any parameters. 😅

Did you add that by mistake or does it point to something that needs to be improved?

Reply:
Yeah, the ref() method doesn't have any parameters and it has to be improved by adding those particular lines of code.